doc/manual/manual.tex

```diff
@@ -1315,14 +1316,233 @@ \subsection{Numerical methods}
+  & \qquad
+  & \textrm{in $\Omega$}.
+\end{align}
+\marginpar{Timo: need to figure out how to solve \eqref{eq:stokes-ALA-2}, need $\nabla \rho$! }
+
+For the temperature equation we arrive at the following:
+\marginpar{Scott: This does not match (18) in \cite{KLKLZTTK10}}
+
+\begin{multline}
+  \label{eq:temperature-decomposed}
+  \bar\rho C_p \left(\frac{\partial T'}{\partial t} + \mathbf u\cdot\nabla
+  T'\right) - \nabla\cdot k\nabla T'
+  \\
+  =
+  \rho H
+  +
+  2\eta
+  \left(\varepsilon(\mathbf u) - \frac{1}{3}(\nabla \cdot \mathbf u)\mathbf 1\right)
+  :
+  \left(\varepsilon(\mathbf u) - \frac{1}{3}(\nabla \cdot \mathbf u)\mathbf 1\right)
+  +\alpha T' \left( \mathbf u \cdot \nabla p \right)
+  \quad
+  \textrm{in $\Omega$}.
+\end{multline}
+Timo: this is not quite correct, you need reference density and simplified adiabatic heating to get:
+\begin{multline}
+  \label{eq:temperature-decomposed2}
+  \bar\rho C_p \left(\frac{\partial T'}{\partial t} + \mathbf u\cdot\nabla
+  T'\right) - \nabla\cdot k\nabla T'
+  \\
+  =
+  \bar\rho H
+  +
+  2\eta
+  \left(\varepsilon(\mathbf u) - \frac{1}{3}(\nabla \cdot \mathbf u)\mathbf 1\right)
+  :
+  \left(\varepsilon(\mathbf u) - \frac{1}{3}(\nabla \cdot \mathbf u)\mathbf 1\right)
+  +\alpha \bar\rho T' (\mathbf u \cdot \mathbf g)
+  \quad
+  \textrm{in $\Omega$}.
+\end{multline}
+
+
+
+\subsubsection{The truncated anelastic liquid approximation (TALA)}
+
+The \textit{Truncated Anelastic Liquid Approximation (TALA)} further simplifies
+the ALA by assuming that the variation of the density due to pressure variations
+is small, i.e., that
+\begin{align*}
+  \rho(p,T) \approx 
+  \bar\rho 
+  + \frac{\partial \rho(\bar p,\bar T)}{\partial T} T'.
+\end{align*}
+This does not mean that the density is not pressure dependent -- it will, for
+example, continue to be depth dependent because the hydrostatic pressure grows
+with depth. It simply means that the deviations from the reference pressure are
+assumed to be so small that they do not matter in describing the density.
+Because the pressure variation is induced by the flow field (the static
+component pressure is already taken care of by the hydrostatic pressure), this
+assumption in essence means that we assume the flow to be very slow, even beyond
+the earlier assumption that we can neglect inertial terms when
+deriving~\eqref{eq:stokes-1}--\eqref{eq:stokes-2}.
+
+This further assumption then
+transforms~\eqref{eq:stokes-ALA-1}--\eqref{eq:stokes-ALA-2} into the following
+equations:
+\begin{align}
+  \label{eq:stokes-TALA-1}
+  -\nabla \cdot \left[2\eta \left(\varepsilon(\mathbf u)
+                                  - \frac{1}{3}(\nabla \cdot \mathbf u)\mathbf 1\right)
+                \right] + \nabla p' &=
+  \frac{\partial \rho(\bar p,\bar T)}{\partial T} T' \mathbf g
+  & \qquad
+  & \textrm{in $\Omega$},
+  \\
+  \label{eq:stokes-TALA-2}
+  \nabla \cdot (\bar\rho \mathbf u) &= 0
+  & \qquad
+  & \textrm{in $\Omega$}.
+\end{align}
```

Comment:
does the TALA have any impact on the temperature equation? If not, maybe say so.